.github/workflows/public-ci.yml

```diff
@@ -58,7 +58,7 @@ jobs:
       run:  sudo apt-get update && sudo apt install libbpf-dev clang llvm gcc-multilib
 
     - name: Install staticcheck
-      run: go install honnef.co/go/tools/cmd/staticcheck@latest
+      run: go install honnef.co/go/tools/cmd/staticcheck@latest && go get github.com/intel/afxdp-plugins-for-kubernetes/pkg/goclient
```

Comment:
Can we separate these two commands. Also, must we import _github.com/intel/afxdp-plugins-for-kubernetes/pkg/goclient_ for fresh environments? if it's required each time, it would be worth adding a check before executing the e2e.